docs/guides/agent-orchestration.md

```
@@ -1,37 +1,56 @@
 
-```markdown
+````markdown
 ### 1.1 Compose another runbook
 
 ```bash
 rd run other.runbook.md
-```
-```
+````
+
+````
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)